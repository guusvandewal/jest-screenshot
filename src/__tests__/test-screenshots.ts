import { setupJestScreenshot } from "..";
import { readFileSync } from "fs";

describe("screenshots", () => {
    beforeEach(() => {
        setupJestScreenshot();
    });

    it("stackoverflow", () => {
        expect(() => {
            expect(readFileSync(`${__dirname}/fixtures/screenshot-stackoverflow.png`)).toMatchImageSnapshot();
        }).toThrowErrorMatchingSnapshot();
    });

    it("npmjs", () => {
        expect(() => {
            expect(readFileSync(`${__dirname}/fixtures/screenshot-npmjs.png`)).toMatchImageSnapshot();
        }).toThrowErrorMatchingSnapshot();
    });
});
