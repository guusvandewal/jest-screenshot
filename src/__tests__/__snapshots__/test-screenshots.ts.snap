// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`screenshots npmjs 2`] = `
"[31mReceived value[39m does not match [32mstored snapshot 1[39m.

Expected less than [32m0.00%[39m of the pixels to have changed, but [31m0.18%[39m of the pixels changed."
`;

exports[`screenshots stackoverflow 2`] = `
"[31mReceived value[39m does not match [32mstored snapshot 1[39m.

Expected less than [32m0.00%[39m of the pixels to have changed, but [31m1.40%[39m of the pixels changed."
`;
